nbr/src/test/java/io/nosqlbench/cli/testing/ExitStatusIntegrationTests.java

```diff
@@ -78,7 +78,7 @@ void testExitStatusOnActivityOpException() {
         invoker.setLogDir("logs/test");
         ProcessResult result = invoker.run("exitstatus_asyncstoprequest", 30,
                 "java", "-jar", JARNAME, "--logs-dir", "logs/test/asyncstop", "--logs-level", "debug", "run",
-                "driver=diag", "cyclerate=10", "op=erroroncycle:erroroncycle=10", "cycles=500", "-vvv"
+                "driver=diag", "threads=2", "cyclerate=10", "op=erroroncycle:erroroncycle=10", "cycles=500", "-vvv"
```

Comment:
Not sure if still relevant but we had this one: `testExitStatusOnActivityThreadException` that was in the older version that would potentially need this threads=2 inclusion.  You may have decided this test was sufficient.  I think the only different was that one threw an unexpected exception instead of using a stop request.  

An `exitstatus_threadexception` I believe.

Reply:
I'll keep an eye on it and apply a work-around if needed. My first inclination is to solve the race condition.